projects/rero/ng-core/src/lib/record/editor/extensions.ts

```diff
@@ -74,11 +74,15 @@ export function getKey(field: any): string {
   }
 
   if (parentKey != null) {
-    let newId = parentKey;
+    const keyParams = [];
     if (field.key != null) {
-      newId = [parentKey, field.key].join('-');
+      keyParams.push(parentKey);
```

Comment:
why only if the field.key is defined? This should be done if the parentKey exists even if the field.key is undefined (oneOf)

Reply:
done